website_event_questions_template/views/website_event_questions_template_view.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<!-- Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+<record id="view_event_question_template_question_form" model="ir.ui.view">
+    <field name="name">event.question.template.question.form</field>
+    <field name="model">event.question.template.question</field>
+    <field name="arch" type="xml">
+        <form string="Event Question Template Question">
+            <sheet>
+                <group>
+                    <field name="sequence"/>
+                    <field name="title"/>
+                    <field name="is_individual"/>
+                </group>
+                <group string="Answers">
+                    <field name="answer_ids" context="{'default_question_id': active_id}"
+                           nolabel="1">
+                        <tree editable="bottom">
+                            <field name="sequence" widget="handle"/>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                </group>
+            </sheet>
+        </form>
+    </field>
+</record>
+
+<record id="view_event_question_template_form" model="ir.ui.view">
+    <field name="name">event.question.template.form</field>
+    <field name="model">event.question.template</field>
+    <field name="arch" type="xml">
+        <form string="Event Question Template">
+            <sheet>
+                <group>
+                    <field name="name"/>
+                </group>
+                <group string="Questions">
+                    <field name="question_ids" nolabel="1"
+                           context="{'default_template_id': active_id}">
+                        <tree string="Questions">
```

Comment:
You shouldn't define an embedded tree, but try to have a primary view inheriting from the other object view. I have 2 doubts with this method:

* Does Odoo allow to define a "cross-model" view?
* Inherited views from the main one are also used in this primary view? I'm pretty that this answer is negative.

Reply:
Are you sure it's worth it?
The view that you say is does an inherit from event... if I do a primary view I have to remove this inheritance....

Reply:
The goal of this is to have the same inherited views as the main one without glue modules. If it's not possible, then don't do it? Do you understand me what I want to achieve?

Reply:
Is this possible then?